google/cloud/internal/populate_common_options.cc

```diff
@@ -58,8 +59,11 @@ Options PopulateCommonOptions(Options opts, std::string const& endpoint_env_var,
     opts.set<UserProjectOption>(*std::move(e));
   }
 
+  if (GetEnv("GOOGLE_CLOUD_CPP_OPENTELEMETRY_TRACING")) {
```

Comment:
Consider also checking that the result is not empty.  That tends to be the general tactic.

Reply:
> Do you think we should be using `ParseBoolean(...)` instead?

Possibly.  (I do have half a PR to add some parsing to getenv.h.)

But mostly I'm thinking about distinguishing between being set with an empty -v- non-empty value.  For example, just a couple of lines up we do that.  (Maybe I'm just too stuck in the past where it was difficult to actually remove something from the environment, so an empty value was usually taken to mean the same thing as unset.)

Anyway, I'll leave it to you.

Reply:
I'm going to treat the presence of the env var as a boolean and ignore its value.

I added a test case for the empty env var to fully enshrine my decision.

Reply:
Nevermind.